Library/Homebrew/cmd/search.rb

```diff
@@ -1,3 +1,21 @@
+#:  * `search` <text>|`/`<text>`/`:
+#:    Perform a substring search of formula names for <text>. If <text> is
+#:    surrounded with slashes, then it is interpreted as a regular expression.
+#:    The search for <text> is extended online to some popular taps.
+#:    If no search term is given, all locally available formulae are listed.
```

Comment:
Not sure where this first block is coming from, but it is duplicating the following two blocks and should probably be removed.